lib/presentation/widgets/m3_widgets/cards/outlined_trip_card.dart

```diff
@@ -110,26 +109,27 @@ class OutlinedTripCard extends StatelessWidget {
             ),
           ),
           if (areThereBottomAction ?? true) const Divider(),
```

Comment:
why you need to `?? true` ?

Reply:
because i use this widget in 2 places- HomePage- with Divider,